src/Apps/Settings/Routes/MyCollection/Components/MyCollectionAppDownload.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Box,
+  Column,
+  Flex,
+  GridColumns,
+  Image,
+  ResponsiveBox,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { resized } from "Utils/resized"
+
+export const MyCollectionAppDownload: React.FC = () => {
+  const image = resized(
+    "https://files.artsy.net/images/my-collection-get-app-img.jpg",
+    {
+      width: 910,
+      height: 652,
+    }
+  )
+
+  return (
+    <GridColumns gridRowGap={4} alignItems="center">
+      <Column span={6}>
+        <Text variant="xxl" mt={1}>
+          Get More from My Collection on the App
+        </Text>
+
+        <Text variant="sm" mt={1} mb={4}>
+          Discover all the features of My Collection on the Artsy app. Coming
+          soon: The full range of features will also be available on the
+          website.
+        </Text>
+
+        <Flex>
+          <Box
```

Comment:
Is there any way we can use a dedicated 'Link' component here?

Reply:
I don't understand. Like the FeaturedLink?

Reply:
Since [Link](https://github.dev/artsy/palette/blob/main/packages/palette/src/elements/Link/Link.tsx#L32) is deprecated we should probably use [RouterLink](https://github.com/artsy/force/blob/33a0b248b68ffb0262d2103d9d1a1c9b85c95abb/src/System/Router/RouterLink.tsx#L28) for links.

Reply:
👀